lib/Sema/DerivedConformances.h

```diff
@@ -216,6 +230,47 @@ class DerivedConformance {
   /// \param synthesizing The decl that is being synthesized.
   bool checkAndDiagnoseDisallowedContext(ValueDecl *synthesizing) const;
 };
+
+/// Create AST statements which convert from an enum to an Int with a switch.
+/// \p stmts The generated statements are appended to this vector.
+/// \p parentDC Either an extension or the enum itself.
+/// \p enumDecl The enum declaration.
+/// \p enumVarDecl The enum input variable.
+/// \p funcDecl The parent function.
+/// \p indexName The name of the output variable.
+/// \return A DeclRefExpr of the output variable (of type Int).
+DeclRefExpr *convertEnumToIndex( SmallVectorImpl<ASTNode> &stmts,
```

Comment:
Please format the declarations in this file according to LLVM style.  You can use `(git) clang-format` to do it automatically.

Reply:
done